.github/workflows/ts-rpc-test.yml

```
@@ -2,37 +2,25 @@ name: TypeScript RPC Client Test
 
 on:
   pull_request:
```

```suggestion
  pull_request:
    paths: ['packages/nitro-rpc-client', 'rpc', '.github/workflows/ts-rpc-test.yml']
```

Is this reasonable? We probably shouldn't be counting on rpc client tests to catch emergent errors inside of go nitro, but this set of paths probably catches any breaking change to the rpc server.

---

I had considered that... but I concluded that we could still break the contract between go-nitro and the TS client without touching `rpc`. So an integration test would break but we wouldn't detect it immediately because it would not run. I'm not 100&% sure though.